tests/Fixtures/inc/Engine/Optimization/RUCSS/Cron/Subscriber/cronCleanRows.php

```diff
@@ -32,20 +32,18 @@
 	'test_data' => [
 		'shouldDeleteOnUpdate' => [
 			'input' => [
-				'remove_unused_css' => true,
-				'used_css'          => $used_css,
+				'used_css' => $used_css,
 				'has_delay' => true,
 				'delay' => 1,
 				'deletion_activated' => true,
 			]
 		],
 		'shouldNotDeleteOnDisabled' => [
 			'input' => [
-				'remove_unused_css' => false,
```

Comment:
@Tabrisrp same here

Reply:
the option is never used for this method